IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_Installation/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_Installation" UniqueId="d81d077a-6f1d-29fc-a8fa-f12640b49c78" ApplicableType="IfcDistributionElement" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="MountingType_3T_MSuVKDEDAQdOQUO_g" />
+		<DocProperty xsi:nil="true" href="ScrewType_1ZaUeczbXDaRBpHqZGoKDX" />
```

Comment:
Thes properties need further explination possibly with the addition of PEnums instead of standard labels.

Additionally would thisdata be covered with other instances within the model such as fittings etc.

Reply:
removed